### ISSUE TITLE
ownCloud storage type

### ISSUE THREAD

**Issue by domenmali** [aloneguid/storage#140]
Hi,

i'm wondering if it would be possible to add ownCloud/nextCloud storage provider?

BR, Domen

**Comment by aloneguid** [comment]
Of course, adding new storage providers is easy enough. Are you willing to contribute with source?

**Comment by domenmali** [comment]
Hi,

some time ago I've wrote a kind of wrapper/helper class that uses AppLimit.CloudComputing.SharpBox library and WebDAV protocol. I can share that code with you if you like.

BR, Domen

**Comment by aloneguid** [comment]
I wasn't aware of Sharpbox, thanks! Please share the code, happy to help.

**Comment by domenmali** [comment]
Hi,

here is the code.

BR, Domen
[WebDAV.Helper.txt](https://github.com/aloneguid/storage/files/3977877/WebDAV.Helper.txt)

**Comment by ivankahl** [comment]
Hi @aloneguid, I use Nextcloud (similar to ownCloud) and use their WebDAV interface so I'm familiar with this and it might help me in the future. 

Can I implement a StorageProvider for WebDAV and submit a PR?

**Comment by aloneguid** [comment]
Of course! That would be a great addition.
